From 428b1b262a122edf88b94ce9e329d7c2fb5af1cb Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 22 Aug 2016 15:11:30 +0300
Subject: [PATCH 23/24] ARM: i.MX6: dts: add hdmi to fb0 mapping for eval board

Add a separate dts that allows hdmi to fb0 mapping for eval board.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6q-sbc-fx6-hdmi.dts | 33 ++++++++++++++++++++++++++++++++
 1 file changed, 33 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6q-sbc-fx6-hdmi.dts

diff --git a/arch/arm/boot/dts/imx6q-sbc-fx6-hdmi.dts b/arch/arm/boot/dts/imx6q-sbc-fx6-hdmi.dts
new file mode 100644
index 0000000..eff992a
--- /dev/null
+++ b/arch/arm/boot/dts/imx6q-sbc-fx6-hdmi.dts
@@ -0,0 +1,33 @@
+/*
+* Copyright 2014 CompuLab Ltd.
+*
+* Author: Valentin Raevsky <valentin@compulab.co.il>
+*
+* The code contained herein is licensed under the GNU General Public
+* License. You may obtain a copy of the GNU General Public License
+* Version 2 or later at the following locations:
+*
+* http://www.opensource.org/licenses/gpl-license.html
+* http://www.gnu.org/copyleft/gpl.html
+*/
+
+/dts-v1/;
+#include "imx6q.dtsi"
+#include "imx6q-cm-fx6.dtsi"
+#include "imx6qdl-sb-fx6x.dtsi"
+#include "imx6qdl-sb-fx6.dtsi"
+
+/ {
+	model = "CompuLab CM-FX6 on SBC-FX6";
+	compatible = "compulab,cm-fx6", "compulab,sbc-fx6", "fsl,imx6q";
+};
+
+&mxcfb1 {
+	mode_str ="1920x1080M@60";
+	disp_dev = "hdmi";
+};
+
+&mxcfb2 {
+	mode_str ="KD050C-WVGA";
+	disp_dev = "lcd";
+};
-- 
1.9.1

